package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "companion",
-	"version": "2.2.0",
```

Comment:
Please don't change the version of core in this pr

Reply:
Sorry, reverted.